akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/RoutersSpec.scala

```diff
@@ -115,6 +113,52 @@ class RoutersSpec extends ScalaTestWithActorTestKit("""
       }
     }
 
+    "support broadcast" must {
```

Comment:
I used `must` because one cannot nest two `in`s one inside the other, however now that I'm removing the 'disabled' test it can be eliminated altogether